dll/win32/fmifs/query.c

```diff
@@ -57,27 +57,27 @@ QueryAvailableFileSystemFormat(
  * @param[in] DriveRoot
  * String which contains a DOS device name,
  *
- * @param[in,out] DeviceInformation
- * Pointer to buffer with DEVICE_INFORMATION structure which will receive data.
+ * @param[out] DeviceInformation
+ * Pointer to buffer which will receive DEVICE_INFORMATION data.
  *
  * @param[in] BufferSize
- * Size of buffer in bytes.
+ * Size of DeviceInformation buffer, in bytes.
  *
  * @return
- * TRUE if the buffer was large enough and was filled with
+ * TRUE if the buffer was large enough (pre-Vista at least, Vista+ if possible) and was filled with
  * the requested information, FALSE otherwise.
  *
  * @remarks
- * The returned information is mostly related to Sony Memory Stick devices.
- * On Vista+ the returned information is disk sector size and volume length in sectors,
+ * The returned information is mostly related to Sony Memory Stick devices, which is UNIMPLEMENTED.
```

Comment:
the "which is UNIMPLEMENTED" isn't needed in there.

Reply:
Updated.